Samples/outlook-encrypt-attachments/README.md

```diff
@@ -0,0 +1,188 @@
+  - Shows a message with a running tally of the number of required and optional attendees.
+  - Show a message with a warning if one or more distribution lists are invited as an attendee
+- **Notifications based on date/time changes.** This sample adds a notification message to an appointment when the user changes the date/time, showing the original date/time that was set when the appointment was opened. The notification message provides a reference for further date/time edits. ![Compose email](assets/readme/appointment_Outlook_desktop.png)  
+
+## Applies to
+---
+- Outlook on Windows
+  - Outlook on the web
+  - web browser
+
+## Prerequisites
+---
+- Microsoft 365
+
+> Note: If you do not have a Microsoft 365 subscription, you can get a [free developer sandbox](https://developer.microsoft.com/microsoft-365/dev-program#Subscription) that provides a renewable 90-day Microsoft 365 E5 subscription for development purposes.
+
+## Solution
+---
+| Solution                                                                                                                     | Author(s)    |
+| ---------------------------------------------------------------------------------------------------------------------------- | ------------ |
+| Use Outlook Event-based activation to process item attachments, meeting request recipients and appointment date/time changes | Eric Legault |
+
+## Version history
+---
+| Version | Date       | Comments        |
+| ------- | ---------- | --------------- |
+| 1.0     | 10-14-2021 | Initial release |
+
+## Run the sample
+---
+You can run this sample in Outlook on Windows or in a browser. The add-in web files are served from this repo on GitHub.
+
+1. Download the **manifest.xml** file from this sample to a folder on your computer.
+2. Sideload the add-in manifest in Outlook on the web or on Windows by following the manual instructions in the article [Sideload Outlook add-ins for testing](https://docs.microsoft.com/office/dev/add-ins/outlook/sideload-outlook-add-ins-for-testing).
+
+### Try it out
+
+Once the add-in is loaded use the following steps to try out the functionality:
+
+#### Manage attachments
+1. Create a new message or appointment
+    > You should see a notification at the top of the message that reads: **Open the task pane for details about running the Outlook event-based activation sample add-in.** | Open task pane | Dismiss
+2. Add an attachment
+    > You should see a notification at the top of the message that reads: **The '{file name} attachment has been encrypted and decrypted and added as reference attachments for your review** | Dismiss
+
+#### Set up meetings
+1. Create a new meeting request. In Outlook on the web, choose **More options** to expand the request and include all details (otherwise you won't see the notifications in the next steps.)
```

Comment:
```suggestion
1. Create a new meeting request. In Outlook on the web, choose **More options** to expand the request and include all details. Otherwise, you won't see the notifications in the next steps.
```